Tax Loss Harvest Alert/src/code.js

```diff
@@ -0,0 +1,59 @@
+/*
+Whenever the spreadsheet changes, 
```

Comment:
Comment and signature is a little confusing for me. Sounds like you're describing this like an onChange or onEdit trigger, but it's not, and the readme suggests setting it up as an time-based trigger.

Reply:
It's actually triggered when there's a change to the spreadsheet - but that happens whenever portfolio stock prices change.  I'll see about clarifying it in the code comment and readme.